lgc/state/PipelineState.cpp

```diff
@@ -63,6 +63,152 @@ static const char RsStateMetadataName[] = "lgc.rasterizer.state";
 static const char ColorExportFormatsMetadataName[] = "lgc.color.export.formats";
 static const char ColorExportStateMetadataName[] = "lgc.color.export.state";
 
+namespace {
+
+// =====================================================================================================================
+// Gets the maximum bit-count of any component in specified color attachment format.
+//
+// @param dfmt : Color attachment data format
+unsigned getMaxComponentBitCount(BufDataFormat dfmt) {
```

Comment:
Please add "static" for those helper functions to restrict their uses in this file.

Reply:
They were restricted because they are in an anonymous namespace.  Should I remove the namespace and add the static or do both?  There is nothing in the coding standard about this.

Reply:
In lgc/patch/PatchCheckShaderCache.cpp, both are used, so that is what I will do.